pkg/utils/constants.go

```diff
@@ -16,9 +16,17 @@ const (
 	DaemonPath                     = "./bindata/manifests/daemon"
 	DefaultPolicyName              = "default"
 	ConfigMapName                  = "device-plugin-config"
+	DaemonSet                      = "DaemonSet"
+	ServiceAccount                 = "ServiceAccount"
 	DPConfigFileName               = "config.json"
 	OVSHWOLMachineConfigNameSuffix = "ovs-hw-offload"
 
 	LinkTypeEthernet   = "ether"
 	LinkTypeInfiniband = "infiniband"
+
+	LinkTypeIB  = "IB"
```

Comment:
should we move this to a new package called Contants?

Reply:
Done